### ISSUE TITLE
Web Service Listener Auth fields should support use of the Global Map

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#2159]
Users should be able to access the global map in the auth settings on the webservice listener using values such as: "$\{userName}"

Imported Issue. Original Details:
Jira Issue Key: MIRTH-2216
Reporter: dschramm
Created: 2012-09-10T13:54:07.000-0700

**Comment by rbeckman-nextgen** [comment]
Revision 7259: The Web Service Listener now supports Velocity replacement for the username/password table.

Imported Comment. Original Details:
Author: narupley
Created: 2014-08-05T08:45:12.000-0700

**Comment by rbeckman-nextgen** [comment]
Created a Webservice listener and added user credentials to the global map in the source transformer. Added the velocity variables that contained these user credentials to the authentication table. I created a webservice sender with using the credentials from the global map. Verified I was able to pull down the wsdls and dispatch a message.

Imported Comment. Original Details:
Author: jaysenp
Created: 2014-08-28T17:52:34.000-0700
